### PR TITLE
Fix total payment

### PR DESCRIPTION
**Pull request by Ismail-ahmadi**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1 +1,30 @@
...
+//   if (radio.checked === true) {
+//     radio.textContent = '$300';
+//     return true;
+//   }
+// }
+// radio.forEach(radio => radio.addEventListener('change', radioPayment));
+// // radio.forEa.addEventListener('click', radioPayment());
+const amount = document.querySelector('.form__total span');
+console.log(amount);
+const inputs = Array.from(document.querySelectorAll('input[type="radio"]'));
```
Comment (nikosolihin) [inline src/main.js:18]: Writing `querySelector` for forms is a pain! So here's a tip:

You can actually just grab the whole `<form>` tag and use the `name=" "` attributes of the containing form elements: 

```javascript
// Test these in devtools console tab
const form = document.querySelector('.form');

// This grabs the radio buttons named plan
form.plan
form.plan.value // Or its selected value

// This grabs the submit button
form.submit

// This grabs the email
form.email
```

File: src/main.js [hunk 2]
```diff
@@ -1 +1,30 @@
...
+
+const annual = document.querySelector('input[value="Annually"]');
+const monthly = document.querySelector('input[value="Monthly"]');
```
Comment (nikosolihin) [inline src/main.js:21]: No need for these if you use my suggestion above...

File: src/main.js [hunk 3]
```diff
@@ -1 +1,30 @@
...
+
+function changeCost() {
+  if (annual.checked === true) {
```
Comment (nikosolihin) [inline src/main.js:24]: ...cause you can just

```javascript
if (form.plan.value === 'Annually') { ... }
```

File: src/main.js [hunk 4]
```diff
@@ -1 +1,30 @@
...
+    amount.textContent = '300';
+  } else {
+    amount.textContent = '30';
+  }
+}
+inputs.forEach(input => input.addEventListener('click', changeCost));
```
Comment (nikosolihin) [inline src/main.js:30]: Btw, `querySelectorAll()` returns a `NodeList` and you can directly run `forEach` on it without converting it into array (`Array.from(NodeList)`).

So this code:

```javascript
const inputs = Array.from(document.querySelectorAll('input[type="radio"]'));
inputs.forEach();
```

can be simpler:

```javascript
const inputs = document.querySelectorAll('input[type="radio"]');
inputs.forEach();
```

But, of course using the dot notation will be simplest:

```javascript
form.plan.forEach()
```

**Review by nikosolihin — CHANGES_REQUESTED** [review]
Great start @Ismail-ahmadi!

Some stuff can be improved — I've left notes.

File: src/main.js (hunk 2 above)
Comment (Ismail-ahmadi) [reply src/main.js]: hahaha yeah these are from my first attempt and forgot to comment out them

**Pull request merged**